### PR TITLE
[MRG] Some aligned 0x20 byte strings had the wrong type.

### PR DESCRIPTION
**Pull request by chrispurnell**

### CODE REVIEW

File: libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+        /* 0x138 */ public NMSString0x10 RidingAnimationName;
+        /* 0x148 */ public bool IsPetCommand;
+        [NMS(Size = 0x7, Ignore = true)]
+        /* 0x149 */ public byte[] Padding149;
+        /* 0x150 */ public NMSString0x20A PetCommandTitle;
+        /* 0x170 */ public TkTextureResource PetCommandIcon;
+        [NMS(Size = 0x4, Ignore = true)]
+        /* 0x1F4 */ public byte[] EndPadding;
+    }
+}
```
Comment (HolterPhylo) [inline libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs:41]: While accurate, any reason why everything is replaced in this structure?

**Comment by HolterPhylo** [comment]
Accurate, I agree!

File: libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs (hunk 1 above)
Comment (chrispurnell) [reply libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs:41]: That happened becuase of line endings.

Comment (monkeyman192) [reply libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs:41]: why does this one random file have a different line ending? So strange...

**Comment by monkeyman192** [comment]
Thanks for doing this! One request though, can you remove any byte padding before aligned strings as they are now not needed and are now just clutter 😄

File: libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs (hunk 1 above)
Comment (chrispurnell) [reply libMBIN/Source/NMS/GameComponents/GcPlayerEmote.cs:41]: Git likes to use lf internally for text files.  Most of the cs files are like that.  But there a few that have been commited without crlf to lf conversion.  That was one of them.

**Comment by chrispurnell** [comment]
There is so much unneeded padding now. I would like to get rid of it all.  Can I do that in another PR?

**Comment by monkeyman192** [comment]
> 
> 
> There is so much unneeded padding now. I would like to get rid of it all. Can I do that in another PR?

yeah sure, I'll merge this now then 😄

**Pull request merged**